pages/multiselect/disabled-reason.page.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as React from 'react';
+import Multiselect, { MultiselectProps } from '~components/multiselect';
+import Box from '~components/box';
+import ScreenshotArea from '../utils/screenshot-area';
+
+const options: MultiselectProps.Options = [
+  { value: 'first', label: 'Simple' },
+  { value: 'second', label: 'With small icon', iconName: 'folder' },
+  {
+    value: 'third',
+    label: 'With big icon icon',
+    description: 'Very big option',
+    iconName: 'heart',
+    disabled: true,
+    disabledReason: 'disabled reason',
+  },
+  {
+    label: 'Option group',
+    options: [{ value: 'forth', label: 'Nested option' }],
+    disabled: true,
+    disabledReason: 'disabled reason',
+  },
+];
+
+export default function MultiselectPage() {
+  return (
+    <article>
```

Comment:
I do not think this element is needed. Same on other pages

Reply:
updated